### PR TITLE
UI auth

### PR DESCRIPTION
**Pull request by huynguyen199**
pull quest with ui-auth
Screenshot_1646934488
Screenshot_1646934469
.

### CODE REVIEW

File: src/screens/auth/containers/formLogin.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+  return (
+    <View style={styles.container}>
+      <Input
+        autoFocus
+        placeholder="Nhập số điện thoại của bạn"
+        inputContainerStyle={styles.inputStyle}
+        value={account}
+        onChangeText={onChangeAccount}
+        containerStyle={
+          !isFocus.acount
```
Comment (troy-tran-goldenowl) [inline src/screens/auth/containers/formLogin.js]: Avoid negative conditional.
```suggestion
          isFocus.acount
           ? styles.inputContainerHover
           : styles.inputContainerStyle
```

File: src/screens/auth/containers/formLogin.js [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+            ? styles.inputContainerStyle
+            : styles.inputContainerHover
+        }
+        onFocus={() => handleInputFocus("acount")}
+        onBlur={() => handleInputBlur("password")}
+      />
+      <Input
+        placeholder="Nhập mật khẩu"
+        rightIcon={
+          !isVisible ? (
```
Comment (troy-tran-goldenowl) [inline src/screens/auth/containers/formLogin.js]: Same as above

File: src/screens/auth/containers/formLogin.js [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+            <EyeIcon onPress={showPasswords} />
+          ) : (
+            <EyeOffIcon onPress={showPasswords} />
+          )
+        }
+        value={password}
+        inputContainerStyle={styles.inputContainerPass}
+        onChangeText={onChangePassword}
+        containerStyle={
+          !isFocus.password
```
Comment (troy-tran-goldenowl) [inline src/screens/auth/containers/formLogin.js]: Same as above

File: src/screens/auth/containers/touchWithoutAccount.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
+import {View, Text, StyleSheet, TouchableOpacity} from "react-native"
+import React from "react"
+import Color from "@common/Color"
+
+const touchWithoutAccount = () => {
```
Comment (troy-tran-goldenowl) [inline src/screens/auth/containers/touchWithoutAccount.js]: Should be TouchWithoutAccount

File: src/screens/auth/fogot.js [hunk 5]
```diff
@@ -0,0 +1,74 @@
+import {View, StyleSheet} from "react-native"
+import React, {useState} from "react"
+import {Header, Icon, Input} from "react-native-elements"
+import Icons from "@common/Icon"
+import Color from "@common/Color"
+import ButtonFogot from "@components/button/index"
```
Comment (troy-tran-goldenowl) [inline src/screens/auth/fogot.js]: Don't need index
```suggestion
import ButtonFogot from "@components/button"
```

**Review by troy-tran-goldenowl — APPROVED** [review]

**Comment by huynguyen199** [comment]
Screenshot_1647224794

**Pull request merged**